### PR TITLE
Pep8

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: lib/bango/resources/package.py [hunk 1]
```diff
@@ -48,8 +48,11 @@ def build_bundle(self, obj=None, data=None, request=None):
     def dehydrate_full(self, bundle):
         if getattr(bundle, 'full', False):
             return response_to_dict(
-                self.client('GetPackage',
-                    {'packageId': bundle.obj.package_id}))
+                self.client(
+                    'GetPackage',
+                    {'packageId': bundle.obj.package_id}
```
Comment (kumar303) [inline lib/bango/resources/package.py]: could probably be on one line

File: lib/bango/tests/test_forms.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ class TestPackage(APITest):
     def test_no_auth(self, clean):
         form = PackageForm(good_address)
         ok_(form.is_valid())
-        ok_(not 'eventNotificationURL' in form.bango_data)
+        ok_('eventNotificationURL' not in form.bango_data)
```
Comment (kumar303) [inline lib/bango/tests/test_forms.py]: hmm. I wonder if this will break (or fix?) anything.

Comment (kumar303) [inline lib/bango/tests/test_forms.py]: oh it's just a test

**Comment by kumar303** [comment]
r+

File: lib/bango/tests/test_forms.py (hunk 2 above)
Comment (andymckay) [inline lib/bango/tests/test_forms.py]: Both work, but the latter is preferred. I fixed a few of these in zamboni and the tests all passed.

**Pull request merged**